.pre-commit-config.yaml

```
@@ -5,7 +5,7 @@ repos:
     hooks:
     - id: missing-django-migrations
       name: Check for missing django migrations
-      entry: ./manage.py makemigrations --no-input --dry-run --check
+      entry: pipenv run python manage.py makemigrations --no-input --dry-run --check
```

```suggestion
      entry: env PIPENV_VERBOSITY=-1 pipenv run python manage.py makemigrations --no-input --dry-run --check
```

Pour pas avoir à se tartiner le warning de pipenv sur le virtualenv.

---

Marrant à moi il ne me dit rien. Le `env` en début de ligne ça change quoi par rapport à juste déclarer la variable ?

---

Juste déclarer la variable fonctionne pas parce que la commande n'est pas exécutée dans un shell. Faut donc appeler une autre commande pour config la variable.